plugins/search-react/src/components/SearchFilter/SearchFilter.tsx

```diff
@@ -69,6 +75,18 @@ export type SearchFilterWrapperProps = SearchFilterComponentProps & {
   debug?: boolean;
 };
 
+/**
+ * @public
+ */
+export interface SelectFilterProps<T extends JsonValue = string>
+  extends SearchFilterComponentProps {
+  multiple?: boolean;
+  emptyItemLabel?: string;
+  placeholder?: string;
```

Comment:
Can `placeholder` and `emptyItemLabel` be ReactNode? So people can wrap it in tags like `<em>` if they want.

Reply:
good point, yes I think, then I guess I will rename `emptyItemLabel` to `allOptionLabel` as suggested above, but word `text` I don't think makes sense in the context of `ReactNode`

Reply:
Nice 💯